styles/ink.less

```diff
@@ -244,3 +244,42 @@ atom-text-editor::shadow {
     font-weight: bold;
   }
 }
+
+.ink-tooltip {
```

Comment:
Yeah, I mean having a tooltip.coffee with its own class and such. May make sense if tooltips are generally reusable.